tests/analysis_tests/tests-generic-jsx-transform/package-lock.json

```diff
@@ -9,6 +9,7 @@
       }
     },
     "../../..": {
+      "name": "rescript",
```

Comment:
I've seen this changing back and forth, maybe depending on `npm` version or something. Best not include these changes in the PR?

Reply:
ok I can remove it, I think latest node versions add it, so I thought we might as well commit and stop seeing this change once for all, but it might be not as consistant as I thought.

Reply:
A single one is still left now.

Reply:
my bad -_-

Reply:
@cknitt fixed!